### ISSUE TITLE
Open valid levels regardless of file extension

### ISSUE THREAD

**Issue by NoelFB** [Ogmo-Editor-3/OgmoEditor3-CE#114]
It would be nice if Ogmo Editor ignored the file extension when opening levels. If my level is named "house.room", and is a valid json level, it should be parsed. Right now it warns that it can't read it.

![image](https://user-images.githubusercontent.com/269893/73598788-80fb9580-44f1-11ea-8f80-2df95c17ef11.png)

I think the file extension could be used as a hint, but if it's not a known type it tries to parse it anyway. If that fails, it then returns the error?

**Comment by AustinEast** [comment]
Sounds good to me! One recommendation i have is that it might be useful to keep a list of extensions to exclude from automatic parsing (things like .mp4, .wav, .png, .jpg, .ogmo, etc), if only to keep the new “Save & Update” button performant enough.

**Comment by NoelFB** [comment]
Yeah that seems like a good idea! Alternatively we could add an option to the Project where you can explicitly add "level file extensions" but that seems like overkill? (especially since this problem might also be unique to me and I don't want to add clutter for a fairly trivial issue).

**Comment by AustinEast** [comment]
I don't think its overkill! it makes sense that a user could want to keep their level file naming scheme separate from whatever other .json files they could have in their project. 

Maybe a "Custom Level Extension" field would do the trick? Then Ogmo would know to load/save to that extension. It'd also be cool if Ogmo could open a .json file, then auto-overwrite it as a .room file (or whatever extension is specified) when the level is saved 😁
